recipes/gklib/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.15)
+project(test_package C)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(GKlib REQUIRED)
```

```suggestion
find_package(GKlib REQUIRED CONFIG)
```

cmake_find_package_multi generates config cmake file, not find cmake file.